src/components/Wizard/CreateVmWizard/StorageTab.js

```diff
@@ -14,140 +14,187 @@ import {
   getDataVolumeResources,
 } from '../../../utils/selectors';
 import { getValidationObject } from '../../../utils/validations';
-
 import { ACTIONS_TYPE, DELETE_ACTION } from '../../Table/constants';
 import {
   ERROR_NO_BOOTABLE_DISK,
   ERROR_EMPTY_ENTITY,
   ERROR_EMPTY_NAME,
-  ERROR_NO_STORAGE_SELECTED,
-  ERROR_NO_STORAGE_CLASS_SELECTED,
   ERROR_POSITIVE_SIZE,
-  ERROR_STORAGE_NOT_VALID,
   ERROR_DISK_NOT_FOUND,
   HEADER_DISK_NAME,
   HEADER_SIZE,
   HEADER_STORAGE_CLASS,
   REMOVE_DISK_BUTTON,
-  ATTACH_STORAGE_BUTTON,
+  ATTACH_DISK_BUTTON,
+  CREATE_DISK_BUTTON,
   BOOTABLE_DISK,
   SELECT_BOOTABLE_DISK,
 } from './strings';
+import { STORAGE_TYPE_PVC, STORAGE_TYPE_DATAVOLUME, STORAGE_TYPE_REGISTRY } from './constants';
 
-const validatePvc = pvc => {
+const validateDataVolumeStorage = storage => {
   const errors = Array(4).fill(null);
```

Comment:
For a follow-up: let's avoid hardcoding of indexes and their manipulation

Reply:
Follow up https://github.com/kubevirt/web-ui-components/issues/131